ur_controllers/CMakeLists.txt

```
@@ -1,35 +1,52 @@
 cmake_minimum_required(VERSION 3.5)
 project(ur_controllers)
 
-add_compile_options(-std=c++11)
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+    set(CMAKE_CXX_STANDARD 14)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+    add_compile_options(-Wall -Wextra)
+endif()
 
 find_package(ament_cmake REQUIRED)
 find_package(controller_interface REQUIRED)
-find_package(hardware_interface REQUIRED)
 find_package(joint_trajectory_controller REQUIRED)
 find_package(pluginlib REQUIRED)
-find_package(realtime_tools REQUIRED)
 find_package(std_msgs REQUIRED)
+find_package(geometry_msgs REQUIRED)
+find_package(rclcpp_lifecycle REQUIRED)
+find_package(rcutils REQUIRED)
 
 set(THIS_PACKAGE_INCLUDE_DEPENDS
-  controller_interface
-  hardware_interface
-  joint_trajectory_controller
-  pluginlib
-  realtime_tools
-  std_msgs
+        controller_interface
+        joint_trajectory_controller
+        pluginlib
+        std_msgs
+        geometry_msgs
+        rclcpp_lifecycle
+        rcutils
```

I am not fond of this stacking of dependencies. It makes the file shorter but less readable.
Still, we can keep it because it was also like this in ROS1.

The old file used two spaces for indentation. let's keep this...
```suggestion
  controller_interface
  joint_trajectory_controller
  pluginlib
  std_msgs
  geometry_msgs
  rclcpp_lifecycle
  rcutils
```